### ISSUE TITLE
Set the concurrency limit as environment variable

### ISSUE THREAD

**Issue by carlochess** [pa11y/pa11y-webservice#83]
Currently, the concurrency limit for the cron job is set to 2 workers. Since in some cases some users don't not need to use too many resources and can get the job done wih only one worker, this number could be set using an environment variable.

_task/pa11y.js_
```
const queue = async.queue(function(task, nextInQueue) {
	console.log('Starting task %s', task.id);
	app.model.task.runById(task.id, function(error) {
		if (error) {
			console.log(chalk.red('Failed to finish task %s: %s'), task.id, error.message);
		} else {
			console.log(chalk.green('Finished task %s'), task.id);
		}
		nextInQueue();
	});
}, config.nWorkers);
```
I can make the change but I was wondering, What do you think about it?

**Comment by joeyciechanowicz** [comment]
I think thats a great idea, thanks for raising this. 

It would need to be named something a bit clearer such as `numWorkers`. And it should default to `2` if the `NUM_WORKERS` env variable is not set, so we are backwards compatible.

**Comment by carlochess** [comment]
@joeyciechanowicz What do you think about this change in the config?
```
if (fs.existsSync(jsonPath)) {
	var jsonConfig = require(jsonPath);

	module.exports = {
		database: env('DATABASE', jsonConfig.database),
		host: env('HOST', jsonConfig.host),
		port: Number(env('PORT', jsonConfig.port)),
		cron: env('CRON', jsonConfig.cron),
		chromeLaunchConfig: jsonConfig.chromeLaunchConfig || {},
		numWorkers: Number(env('NUM_WORKERS', '2'))
	};
...
```

**Comment by carlochess** [comment]
Reference [PR-84](https://github.com/pa11y/pa11y-webservice/pull/84)

**Comment by josebolos** [comment]
Fixed by #84.
